global/install/skm_global_install.sh

```diff
@@ -89,7 +155,12 @@ fi
 
 if [ "$SK_OS" = "linux" ]; then
     echo "Updating library config cache"
-    $PRIVILEGED ldconfig /usr/local/lib
+    # Add /usr/local/lib to ld search paths using conf file
+    touch "${SKM_PATH}/linux/splashkit.conf"
+    echo "/usr/local/lib" >> "${SKM_PATH}/linux/splashkit.conf"
+    $PRIVILEGED cp -n "${SKM_PATH}/linux/splashkit.conf /etc/ld.so.conf.d/splashkit.conf"
```

Comment:
What about `mv` rather than `cp` here? Then no need for the `rm`

Reply:
Totally forgot about `mv` 😅 fixed now.